.eslintrc.json

```diff
@@ -88,10 +88,7 @@
     "key-spacing": "off",
     "keyword-spacing": "off",
     "line-comment-position": "off",
-    "linebreak-style": [
-      "error",
-      "unix"
-    ],
+    "linebreak-style": "off",
```

Comment:
Do we want consistent unix-style linebreaks across all platforms?

What if instead of disabling the line we introduce a `.gitattributes` file in the root of the monorepo containing:

```
* text=auto
```

We should then be able to keep the linebreak-style check in place. That way, if a windows style line break sneaks in, the linter will catch it.

See: https://help.github.com/en/articles/dealing-with-line-endings

I could be wrong and this approach won't work, but atm I don't have my windows laptop with me to experiment.

Reply:
A `.gitattributes` file has been introduced, which seems like a good idea based on the github link in my previous comment. However, it doesn't prevent eslint from barfing on what AzP checks out. I think it's related to: https://github.com/Microsoft/azure-pipelines-yaml/issues/11.